Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1258,13 +1258,26 @@ inline void BuilderCustom::applyValueContent(BuilderState& builderState, CSSValu
             builderState.style().setHasAttrContent();
         else
             const_cast<RenderStyle&>(builderState.parentStyle()).setHasAttrContent();
-        QualifiedName attr(nullAtom(), primitiveValue.stringValue().impl(), nullAtom());
+
+        auto value = primitiveValue.cssAttrValue();
+        QualifiedName attr(nullAtom(), value->attributeName().impl(), nullAtom());
         const AtomString& attributeValue = builderState.element() ? builderState.element()->getAttribute(attr) : nullAtom();
 
         // Register the fact that the attribute value affects the style.
         builderState.registerContentAttribute(attr.localName());
 
-        return attributeValue.isNull() ? emptyAtom() : attributeValue.impl();
+        if (attributeValue.isNull()) {
+            auto fallback = value->fallback();
+
+            if (fallback != nullptr) {
+                auto fallbackValue = downcast<CSSCustomPropertyValue>(fallback)->customCSSText();
+                return downcast<CSSCustomPropertyValue>(fallback)->customCSSText().substring(1, fallbackValue.length() - 2).impl();
```

Comment:
Just use `fallback->cssText()`?

Reply:
See my other comment, we should have used `CSSParserImpl::consumeDeclarationValue`